shared/chat/conversation/messages/system-change-retention/index.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import moment from 'moment'
+import * as React from 'react'
+import * as Kb from '../../../../common-adapters'
+import * as Styles from '../../../../styles'
+import UserNotice from '../user-notice'
+import {formatTimeForMessages} from '../../../../util/timestamp'
+import * as RPCChatTypes from '../../../../constants/types/rpc-chat-gen'
+
+type Props = {|
+  canManage: boolean,
+  isTeam: boolean,
+  isInherit: boolean,
+  membersType: RPCChatTypes.ConversationMembersType,
+  onClickUserAvatar: (username: string) => void,
```

Comment:
why bother having a username if its just the props.username. just make it () and pass the username in mergeProps